backend/python_init_script/main.py

```diff
@@ -2,41 +2,186 @@
 # SPDX-FileCopyrightText: 2021 Felix Steinkohl <steinkohl@campus.tu-berlin.de>
 
 import os
+from dataclasses import dataclass
+import pandas as pd
 from appwrite.client import Client
 from appwrite.services.users import Users
 from appwrite.services.teams import Teams
+from appwrite.exception import AppwriteException
+
+
+@dataclass
+class UserData:
+    """
+    Dataclass for specifying user data
+    """
+
+    name: str
+    email: str
+    team: str
+    roles: [str]
+    init_pw: str
+
+
+def load_user_data(file: str) -> [UserData]:
+    """
+    Reads the given file and parses it into UserData objects.
+
+    :param file: path of file containing the user data that shall be loaded
+    :type file: str
+    :return: iterable of the loaded UserData objects
+    :rtype: [UserData]
+    """
+    ending = str(file).lower().split(".")[-1]
+    user_df = pd.DataFrame()
```

Comment:
I'm kind of divided how I would rate `user_df` as a name. On the one hand, "dataframe" is a pretty meaningless addition and something like `user_record` would better fit. On the other hand, it's kind of like hungarian notation and it tells you the expected datatype which is fine for Python.

Btw, is it legitimate to write this line in the `else` branch?